db/migrate/20180509061518_create_patients.rb

```diff
@@ -0,0 +1,11 @@
+class CreatePatients < ActiveRecord::Migration[5.1]
+  def change
+    create_table :patients, id: false do |t|
+      t.uuid :id, primary_key: true
```

Comment:
What's the reason for using UUID instead of integers?

Reply:
All the records are created on devices, and sync'd across devices. Using UUIDs makes it easy to track the same patient across devices. Using serial ids would inevitably cause collisions. For performance reasons, UUIDs are stored and treated like large numbers, and should be fairly performant.

We aren't actually sure how this would work using only integers. Did you have something in mind?

-- Srihari and Govind

Reply:
Good point; I wasn't considering collisions.